src/test/scala/org/ergoplatform/api/routes/InfoRoutesSpec.scala

```diff
@@ -22,7 +24,8 @@ class InfoRoutesSpec extends FlatSpec
   it should "return info" in {
     Get("/info") ~> route ~> check {
       status shouldBe StatusCodes.OK
-      InfoRoute.makeInfoJson(nodeId, minerInfo, connectedPeers.length, readers, "digest").toString shouldEqual responseAs[String]
+      val resp = responseAs[String]
+      Info.make(nodeId, minerInfo, connectedPeers.length, readers, "digest").asJson.toString shouldEqual resp
```

Comment:
This doesn't verify that the fix has been made. It will verify that the same output was produced using two different functions which under the hood use the same machinery. 

Proper test would be to check that `difficulty` field is not a `Double` in any case. I would suggest to use `scalacheck` for generating the sample data and check for `.` existence.

Reply:
Yeah. You are right. Will do it.

Reply:
Also I'm thinking that now as we convert `Long` to `JSON`, and this conversion is determined, there is no need to check it. Cause it's circe responsibility to check this case.